src/modules/Hosts/Hosts/Views/MainPage.xaml

```diff
@@ -37,25 +37,23 @@
             TrueValue="Collapsed" />
     </Page.Resources>
 
-    <Grid>
+    <Grid
+        Margin="16"
+        RowSpacing="8">
         <Grid.RowDefinitions>
-            <RowDefinition Height="64" />
-            <RowDefinition Height="*" />
```

Comment:
Why removing a row?
The file changed and save failed info bars are now overlapping the hosts list.

`0.71 / PR`

![image](https://github.com/microsoft/PowerToys/assets/25966642/e5751f29-b49c-4077-8f0f-9438309d2a31)

Reply:
Ah yes, you got me there. The infobars were kind of "invisible" to me. I'll revert.

Reply:
Now, I'm thinking it might also be good to remove the question from the text.